### PR TITLE
[BD-26] Timer bar on non-sequence pages

### PR DESCRIPTION
**Pull request by UvgenGen**
Timer bar on non-sequence pages

**Jira**: [EDUCATOR-5857](https://openedx.atlassian.net/browse/EDUCATOR-5857)
NOTE: need merge with https://github.com/edx/frontend-app-learning/pull/502 and https://github.com/edx/edx-proctoring/pull/894

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @UvgenGen! I've created [BLENDED-898](https://openedx.atlassian.net/browse/BLENDED-898) to keep track of it in Jira.   More details are on the [BD-26](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2523627980/BD-26+Proctoring+MFE+and+configuration) project page.


When this pull request is ready, tag your edX technical lead.

**Review by idegtiarov — APPROVED** [review]
LGTM

File: src/exam/ExamWrapper.jsx [hunk 1]
```diff
@@ -36,12 +36,16 @@ const ExamWrapper = ({ children, ...props }) => {
 
 ExamWrapper.propTypes = {
   sequence: PropTypes.shape({
-    id: PropTypes.string.isRequired,
+    id: PropTypes.string,
```
Comment (Zacharis278) [inline src/exam/ExamWrapper.jsx:39]: this can be marked as required now with no default props right?

**Review by Zacharis278 — CHANGES_REQUESTED** [review]

File: src/exam/ExamWrapper.jsx (hunk 1 above)
Comment (ViktorRusakov) [reply src/exam/ExamWrapper.jsx]: yeah, reverted it in a229462

**Review by Zacharis278 — APPROVED** [review]

**Pull request merged**